efcore-migration-task/efcore-migration-script-generator/index.ts

```diff
@@ -58,7 +58,7 @@ async function run() {
                             .arg('tool')
                             .arg('install')
                             .arg('--global')
-                            .arg('dotnet-ef');
+                            .arg('dotnet-ef --version 3.0.0-*');
```

Comment:
Could you change this to be three lines instead? Like

    .arg('dotnet-ef');
    .arg('--version');
    .arg('3.0.0-*');

Reply:
Yes, i will update right away. For the benefits of this: It will help install the latest version of EF Core global tool whenever installation of dependencies is set true. This will be very helpful since the tools is being actively updated.

Reply:
Doesn’t the current version install the latest version already? If `--version 3.0.0-*` is added, I don’t version 3.1.0.0 will be installed when that is released.

Reply:
You have to update for major releases. this fix is for the minor release and will be very helpful for breaking changes.

Reply:
I’m still thinking about what to do with this. You are absolutely right that something needs to be done to be prepared with future updates. I will probably wait until .NET 3.1 is released to get a feeling what needs to be done. I’m thinking about adding an option where the user could select which version to be installed.

Reply:
@pekspro I can see you have added a new option for **Install dependency for .NET Core 3.0** but it's disable. Could you please post the syntax of the input like this:

- task: efcore-migration-script-generator-task@0
  inputs:
    projectpath: '.....Context.csproj'
    databasecontexts: '....'
    startupprojectpath: '....csproj'
    targetfolder: '$(build.artifactstagingdirectory)/Migrations'
    **new input: '?'**

Reply:
@ashkanparsa, I think `installdependencies: 'true'` is what you are looking for (I haven't tried myself).